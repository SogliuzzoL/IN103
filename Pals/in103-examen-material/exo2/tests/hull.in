4
3.0, 2.5
2.5, 4.5
2.0, 1.0
4.5, 1.5
