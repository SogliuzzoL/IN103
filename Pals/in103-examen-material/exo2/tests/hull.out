(2.000000, 1.000000)
(2.500000, 4.500000)
(4.500000, 1.500000)
